five_tips_about_testable_golang_code.md

```diff
@@ -0,0 +1,171 @@
+# How to write untestable code in golang
+According to my experience in movies, if you want to be a super bad guy, you will need to work harder than the good guys. Think about it! You need to hire evil scientists, build huge death bomb to threaten the world, a good plan and a lot of money. But be a good guy basically just need some brave hearts, you do not even have known about the structure of bomb because when choosing red/blue wire to cut, good guys never mess up.
```

Comment:
-build a huge death bomb, or build huge death bombs?
-But to be a good guy you just basically?
-you do not even have to know about?